### ISSUE TITLE
Android devices Screens fading out even in full screen mode

### ISSUE THREAD

**Issue by CarloCattano** [Cycling74/miraweb#39]
For some reason it does not recognize the fullscreen command as "keep the screen activated", as expected and experienced in any videoplayer etc

**Comment by fde31** [comment]
Just to clarify/double check: you're saying that the screen goes off compared to video playback when in fullscreen mode?

A quick googling showed that there is a [Wake Lock API](https://w3c.github.io/wake-lock/) proposal but I couldn't find immediate info wrt available implementations in certain browsers and various platforms but that might be a good way to start looking into this.

Apart from that just a few initial things that come to mind:

One might wanna verify but on iOS it's possible to prevent ScreenLock entirely via the OS settings. I'd have to double check on an Android device if a similar setting/feature is available. I remember dealing with this once and solving it either via a 3rd party app or an Advanced Developer Setting on the device that prevented sleep when charging. But guess all of these aren't 100% satisfying.

If the Wake Lock API isn't sufficient there seem to be some workarounds like NoSleep.js in the wild that might be worth investigating.

**Comment by CarloCattano** [comment]
yes , in Android is possible to do as you explain . so it is possible but less convenient i guess. thanks 
also orientation must be locked , otherwise when using accelerometer input screen change affects orientation reading , so we are used to suffer under android dont worry too much about it . It was just a consideration and if there it was an easy option to try implementing it without much complications

**Comment by CarloCattano** [comment]
Fullscreen maybe could do the trick too . And its quite usefull actually , since for most use cases you dont wantto see the browser tab, and perhaps that will also allow to lock rotation . Didnt try but I will soon following this reference https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API
